.gitignore

```diff
@@ -26,7 +26,7 @@ cms
 .idea
 .DS_Store
 
-coverage.txt
+coverage.out
```

Comment:
?

Reply:
it fixes codecov

Reply:
It's confusing to have an unrelated change like that in this PR. I suggest creating a new PR to just fix codecov, and get that merged in first

Reply:
It's just one line

Reply:
@fqutishat I also see changes to the `check_unit.sh` script too... so not just one line...

At the very least, adding "also fixes codecov" or something similar to the commit body message will help future people understand why those lines were changed.

Reply:
Having the separate commit also makes it easier to figure out what fixed what. e.g. when you're in the IDE and you use the annotate feature, it'll show that line changed and when you click on it, you can see a commit that describes exactly what happened "fix: codecov". With the change in this commit, you see something confusing: "feat: configure database connection retries"

Reply:
Adding a "also fixes codecov" to the commit body message is good enough for me

Reply:
fixed